desktop/notification/fdo.go

```diff
@@ -180,10 +219,15 @@ func processNotificationClosed(sig *dbus.Signal, observer Observer) error {
 	if !ok {
 		return fmt.Errorf("expected second body element to be uint32, got %T", sig.Body[1])
 	}
-	return observer.NotificationClosed(ID(id), CloseReason(reason))
+	if localID, ok := srv.serverToLocalID[id]; ok {
```

Comment:
could you add a comment that we expect to receive method calls for notifications we didn't send and thus it's the correct behavior here to just silently ignore those ?